packages/apps/lightning-ui-docs/index.js

```diff
@@ -23,7 +23,7 @@ import {
   Speech,
   pool,
   context
-} from '@lightningjs/ui-components';
+} from '@lightningjs/ui-components/src';
```

Comment:
still unsure on why we need src here (if these things aren't being exported from the root, maybe they should be)

Reply:
same as above

Reply:
see comment on ThemeDownload.js